chapters/classes.tex

```diff
@@ -644,20 +644,20 @@ \subsection{Short Class Definitions}\label{short-class-definitions}
 
 A class definition of the form
 \begin{lstlisting}[language=modelica]
-class IDENT1 = IDENT2 class-modification;
+class IDENT1 = type-specifier class-modification;
 \end{lstlisting}
 
-is identical, except that IDENT2 may be replaceable and for the lexical
+is identical, except that \lstinline!type-specifier! (the base-class) may be replaceable and for the lexical
 scope of modifiers, where the short class definition does not introduce
 an additional lexical scope for modifiers, to the longer form
 
```

Comment:
Empty line causing bad paragraph break:
```suggestion
```

(If one really wants a paragraph break at a place like this, it needs to be explained why in a comment.)